src/BinaryParsers/PEBinary/ProgramDatabase/CompilerCommandLine.cs

```diff
@@ -118,6 +130,13 @@ public CompilerCommandLine(string commandLine)
                             // https://docs.microsoft.com/cpp/build/reference/o-options-optimize-code?view=msvc-170
                             // /O1 /O2 /Og /Os /Ot /Ox are all indicative of optimizations being enabled
                             this.OptimizationsEnabled = true;
+
+                            if (argument.EndsWith("O1") || argument.EndsWith("O2"))
+                            {
+                                // https://docs.microsoft.com/cpp/build/reference/gf-eliminate-duplicate-strings?view=msvc-170#remarks
+                                // /GF is in effect when /O1 or /O2 is used.
+                                this.EliminateDuplicateStringsEnabled = true;
```

Comment:
If I understand this correctly, O1/O2 indicate `GF` plus eliminate duplicate strings? First read through, I thought this indicated a duplicate check for GF was being introduced. Since we have so many people involved in this code base, it's helpful to be very clear with the comments to prevent someone trying to be "helpful" and removing a check in the future.

Reply:
I tried to clarify the comment.  Please let me know if it makes more sense now.